src/megachatapi_impl.cpp

```diff
@@ -1813,6 +1813,39 @@ void MegaChatApiImpl::sendPendingRequests()
             fireOnChatRequestFinish(request, megaChatError);
             break;
         }
+        case MegaChatRequest::TYPE_ENABLE_AUDIO_LEVEL_MONITOR:
+        {
+            handle chatid = request->getChatHandle();
+            bool enable = request->getFlag();
+            if (chatid == MEGACHAT_INVALID_HANDLE)
+            {
+                API_LOG_ERROR("MegaChatRequest::TYPE_ENABLE_AUDIO_LEVEL_MONITOR - Invalid chatid");
+                errorCode = MegaChatError::ERROR_NOENT;
```

Comment:
I'd say an invalid handle is an invalid arguments (`ERROR_ARGS`), but it's true that we already have `ERROR_NOENT` in multiple cases like this already. Up to you :)

Reply:
I use `ERROR_NOENT` because I observed that it was used in more case but I think it's more correct `ERROR_ARGS`
I've set `ERROR_ARGS`